extensions/styled_components.yaml

```diff
@@ -36,8 +36,8 @@ contexts: !merge
                 captures:
                   0: punctuation.definition.template-expression.begin.js
                 push:
-                  - clear_scopes: 1
-                  - meta_scope: meta.template.expression.js
+                  - clear_scopes: true
+                  - meta_scope: soucre.js.react meta.template.expression.js
```

Comment:
Typo.

Reply:
Also, we shouldn't hardcode the top-level scope as `source.js.react`, because the top-level scope is determined by the configuration. Its value is in the `scope` argument, so we should be able to do something like `!format '{scope} meta.template.expression.js'`.

Reply:
Thanks for giving me a chance. I made the requested changes.

Reply:
Sorry for the typo. I did a quick test after changes and it works flawlessly now.